test/parallel/process-no-deprecation.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+// Flags: --expose_internals
+process.noDeprecation = true;
+
+require('../common');
+const assert = require('assert');
+
+process.on('warning', (warning) => {
+  assert.notStrictEqual(warning.name, 'DeprecationWarning');
+});
+
+const internalUtil = require('internal/util');
+internalUtil.printDeprecationMessage('Something is deprecated.');
```

Comment:
Can you add a check that calling `printDeprecationMessage()` after setting `process.noDeprecation` to `false` again does emit a deprecation warning?